functions.php

```diff
@@ -177,22 +185,23 @@ function vf_validate_options($options) {
  * Validate that the provided url is a vanilla forum root. Returns properly formatted url if it is, or FALSE.
  */
 function vf_validate_url($url) {
```

Comment:
You de-optimized that function.
The function was returning a result as soon as it reached `vf_parse_webroot()`.
Now it will do further check for nothing.
A `break` could solve that but that function was fine.